components/browser/engine-system/src/main/res/values-uz/strings.xml

```diff
@@ -2,6 +2,9 @@
 <resources>
     <!-- Text for the title of an alert dialog displayed by a web page. %1$s will be replaced with the URL of the current page (displaying the dialog). -->
     <string name="mozac_browser_engine_system_alert_title">%1$s sahifasi xabari:</string>
+    <!-- Text for the message of an auth dialog displayed by a web page.
+    %1$s will be replaced by the hostname or a description of the protected area/site, %2$s will be replaced with the URL of the current page (displaying the dialog). -->
+    <string name="mozac_browser_engine_system_auth_message">%1$s login va parolingizni soʻrayapti. Sayt xabari: “%1$s”</string>
```

Comment:
```suggestion
    <string name="mozac_browser_engine_system_auth_message">%2$s login va parolingizni soʻrayapti. Sayt xabari: “%1$s”</string>
```

@flodolo looks like the placeholder is the only part that is incorrect based on the string below that is similar.

Reply:
Too bad I already fixed this [2 months ago](https://pontoon.mozilla.org/uz/android-l10n/mozilla-mobile/android-components/components/browser/engine-system/src/main/res/values/strings.xml/?search=mozac_browser_engine_system_auth_message&string=198458) :-\

It should be fixed again.

@uzbadmin 
Please don't revert this string again, and check the comment associated to the old translation.